packages/front-end/components/Experiment/ExperimentDateGraph.tsx

```diff
@@ -16,9 +16,12 @@ import { date } from "shared/dates";
 import { variant_0, variant_1, variant_2, variant_3 } from "shared/constants";
 import { StatsEngine } from "back-end/types/stats";
 import { pValueFormatter } from "@/services/experiments";
+import { srmHealthCheck } from "../HealthTab/SRMDrawer";
+import { BadgeColors } from "../HealthTab/HealthDrawer";
 import styles from "./ExperimentDateGraph.module.scss";
+import { SRM_THRESHOLD } from "./SRMWarning";
```

Comment:
Does this need to be the `DEFAULT_SRM_THRESHOLD` from `/front-end/settings/index.tsx`?